app/services/company_member_events_discriminator.rb

```diff
@@ -0,0 +1,15 @@
+class CompanyMemberEventsDiscriminator < BaseService
+  def initialize(events)
+    @events = events
+  end
+
+  def call
+    @events.reject { |event| company_members.include? event[:repo][:name].split('/').first }
+  end
+
+  private
+
+  def company_members
+    User.pluck(:login) << ENV.fetch('GITHUB_ORGANIZATION', 'rootstrap')
```

Comment:
I'd remove the default. Remember this is an OSS project, so I'd like to leave no references to rootstrap